bitnami/openldap/2.6/debian-12/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -53,6 +53,7 @@ export LDAP_DAEMON_GROUP="slapd"
 export LDAP_PORT_NUMBER="${LDAP_PORT_NUMBER:-1389}"
 export LDAP_LDAPS_PORT_NUMBER="${LDAP_LDAPS_PORT_NUMBER:-1636}"
 export LDAP_ROOT="${LDAP_ROOT:-dc=example,dc=org}"
+export LDAP_SUFFIX="$(if [ -z ${LDAP_SUFFIX+x} ]; then echo ${LDAP_ROOT}; else echo ${LDAP_SUFFIX}; fi)"
```

Comment:
Could you please quote the variables?

```suggestion
export LDAP_SUFFIX="$(if [ -z "${LDAP_SUFFIX+x}" ]; then echo "${LDAP_ROOT}"; else echo "${LDAP_SUFFIX}"; fi)"
```